pastis/config_pastis.ini

```diff
@@ -5,7 +5,11 @@
 [local]
 ; figure out webbpsf-data path with: webbpsf.utils.get_webbpsf_data_path()
 webbpsf_data_path = /Users/<user-name>/anaconda/envs/astroconda/share/webbpsf-data
-local_data_path = /Users/<user-name>/data_from_repos/pastis_data
+#local_data_path = /Users/<user-name>/data_from_repos/pastis_data
+
+local_data_path = /Users/asahoo/Desktop/data_repos/laurent_data
+local_repo_path = /Users/asahoo/laurent_repos/PASTIS
+output_data_folder = /Users/asahoo/Desktop/data_repos/laurent_data
```

Comment:
Please don't change the default paths in the config_ini.ini since that will mess things up. Your local paths should be set in the config_local.ini only.

Reply:
@ananyasahoo1904 here